ArkSavegameToolkitNet/ArkSaveGame.cs

```diff
@@ -126,6 +130,84 @@ public bool LoadEverything()
+                    if (byteProperty != null)
+                {
+        
+                        StructPropertyList byteArrayProperty = (StructPropertyList)byteProperty.Value;
+                        StructPropertyList byteArrayList = (StructPropertyList)byteArrayProperty.GetProperty<PropertyArray>("ByteArrays").Value[0];
+                        var byteList = byteArrayList.GetProperty<PropertyArray>("Bytes");
+
+                        //convert from sbytes to bytes 
+                        sbyte[] sbyteValues = new sbyte[byteList.Value.Count];
+                        for (int x = 0; x < byteList.Value.Count; x++)
+                        {
+                            sbyteValues[x] = (sbyte)byteList.Value[x];
+                        }
+                        byte[] cryoDataBytes = (byte[])(Array)sbyteValues;
+
+                        //easier to save to disk and read back as memorymappedfile
+                        string filePath = _fileName.Substring(0, _fileName.LastIndexOf(".")) + ".cryo";
+                        File.WriteAllBytes(filePath, cryoDataBytes);
+                        var fi = new FileInfo(filePath);
+                        var size = fi.Length;
+
+                        using (MemoryMappedFile cryoMmf = MemoryMappedFile.CreateFromFile(filePath, FileMode.Open, null, 0L, MemoryMappedFileAccess.Read))
+                        {
+                            using(MemoryMappedViewAccessor cryoVa = cryoMmf.CreateViewAccessor(0L, 0L, MemoryMappedFileAccess.Read))
+                            {
+                                ArkArchive cryoArchive = new ArkArchive(cryoVa, size, _arkNameCache, _arkStringCache, _exclusivePropertyNameTree);
+                                cryoArchive.GetBytes(4);
+
+                                nextObjectId++;
+                                var dino = new GameObject(cryoArchive);
+                                dino.ObjectId = nextObjectId;
+                                dino.IsCryo = true;
+
+                                nextObjectId++;
+                                var statusobject = new GameObject(cryoArchive);
+                                statusobject.ObjectId = nextObjectId;
+
+                                dino.loadProperties(cryoArchive, new GameObject(), 0, null);
+                                var statusComponentRef = dino.GetProperty<PropertyObject>("MyCharacterStatusComponent");
+                                statusComponentRef.Value.ObjectId = statusobject.ObjectId;
+                                dino.properties.Remove(_myCharacterStatusComponent);
+                                dino.properties.Add(_myCharacterStatusComponent, statusComponentRef);
+
+                                statusobject.loadProperties(cryoArchive, new GameObject(), 0, null);
+
+                                Objects.Add(dino);
+                                Objects.Add(statusobject);
+
+                                cryoVa.Dispose();
```

Comment:
may not cause an error, but the using block will implicitly call Dispose on variable when leaving execution.. so an explicit call is redundant or has the potential to cause an error on types that don't handle dispose properly (not likely the case here)